controllers/apply/form-router-next/joi-extensions/phone-number.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const { PhoneNumberUtil, PhoneNumberFormat } = require('google-libphonenumber');
+const phoneUtil = PhoneNumberUtil.getInstance();
+
+/**
+ * Allows you to do `Joi.string().phoneNumber()`
+ *
+ * @param {Object} joi Joi instance
+ * @return {Object} Joi plugin object
+ */
+module.exports = function phoneNumber(joi) {
+    return {
+        base: joi.string(),
+        name: 'string',
+        language: {
+            phonenumber: 'did not seem to be a phone number'
+        },
+        rules: [
+            {
+                name: 'phoneNumber',
+                validate(params, value, state, options) {
+                    try {
+                        const parsedValue = phoneUtil.parse(value, 'GB');
```

Comment:
We only care about uk numbers so can avoid the abstraction of supporting configurable options by hardcoding.

Reply:
+1